Makefile

```
@@ -46,6 +46,11 @@ clean:
 	rm -rf .build/
 
 
+install-git-hooks:
+
+	chmod +x .githooks/pre-push
+	git config core.hooksPath .githooks
```

```suggestion
	git config --local core.hooksPath .githooks
```
Would it be better to add `--local` to avoid unintended changes to other repositories?